<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parametros</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>641</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_cerrar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>400</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>png/001-cancel.png</normaloff>png/001-cancel.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>RUTA DE DESCARGA</string>
     </property>
     <widget class="QPushButton" name="btn_ruta">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Escoger ruta</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scroll">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>39</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>229</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>401</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>RANGO DE FECHA</string>
     </property>
     <widget class="QDateEdit" name="fecha_to">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="fecha_from">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Desde la fecha</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hasta la fecha</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_ok">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>400</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>png/002-check.png</normaloff>png/002-check.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>401</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>EMITIDAS - RECIBIDAS</string>
     </property>
     <widget class="QComboBox" name="combo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
